src/main/resources/swagger/agora.yaml

```diff
@@ -909,17 +974,17 @@ definitions:
         type: string
         description: Payload of method -- must be in WDL format
         default: |
-          task wc { command { cat ${File in_file} | wc -l } output { Int count = read_int(stdout()) }}
+          { "methodStoreMethod": { "methodNamespace": "YOUR_NAMESPACE", "methodName": "BWA", "methodVersion": 1}}
       entityType:
         type: string
-        description: Tyoe of the AgoraEntity -- Task or Workflow.
+        description: Type of the AgoraEntity -- Task or Workflow.
         default: Task
   MethodResponse:
```

Comment:
For both methods and configurations, payload and description are excluded by default in the return from the list method. (they are included when getting single entity). However, you CAN request them while doing a list by explicitly adding them to includedFields. Is there any way to make this clear?